antarest/study/storage/variantstudy/model/command/create_link.py

```diff
@@ -30,8 +31,8 @@ class LinkProperties:
     LOOP_FLOW: bool = False
     USE_PHASE_SHIFTER: bool = False
     DISPLAY_COMMENTS: bool = True
-    TRANSMISSION_CAPACITIES: str = "enabled"
-    ASSET_TYPE: str = "ac"
+    TRANSMISSION_CAPACITIES: TransmissionCapacity = "enabled"
```

Comment:
I think it should be `TransmissionCapacity.ENABLED` and `AssetType.AC`. It shushes mypy issues